app/src/androidTest/java/com/hedvig/app/util/EspressoCustomMatcher.kt

```diff
@@ -0,0 +1,20 @@
+package com.hedvig.app.util
+
+import android.view.View
+import androidx.test.espresso.matcher.BoundedMatcher
+import com.agoda.kakao.bottomnav.KBottomNavigationView
+import com.google.android.material.bottomnavigation.BottomNavigationView
+import org.hamcrest.Description
+import org.hamcrest.Matcher
+
+fun KBottomNavigationView.hasNumberOfMenuItemsCheck(matcherNumber: Int): Matcher<View?>? {
```

Comment:
The function can live in the `KakaoExtensions.kt`-file, but the class should be extracted and named (instead of being an anonymous class in the form of `object: BoundedMatcher...`)